source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -336,12 +336,6 @@ The following items must be taken care of to allow this:
 
 ## Further considerations
 
-### IPAM
-
-The IP assigned to OVN managed NICs should be assigned from a subnet (pool of IPs) definded within OVN.
```

Comment:
please keep this section. Just change the tense from future to present

"The IP assigned to OVN managed NICs *is* assigned from a subnet"...